CafeTime/CafeTime/StringsChecker.swift

```diff
@@ -15,10 +15,10 @@ class StringsChecker {
     
     // MARK: Pulic Funcs
     
-    // login details
+    // login credentials details
     func checkLoginDetails(email : String, password: String) -> String {
```

Comment:
you can change return type to "String?" and in a case of error return errorString. if no error, just simple return without "" (line 50)